src/webui/static/app/idnode.js

```diff
@@ -2086,16 +2105,21 @@ tvheadend.idnode_grid = function(panel, conf)
 
         plugins.push(filter);
         var gconf = {
+            id: 'idnode_grid',
```

Comment:
Do not hardcode anything here - use conf.id or so.. Perhaps, this change is not required at all - 'select' might contain a pointer to the column model in the callback.